scene/resources/resource_format_text.cpp

```diff
@@ -1398,6 +1398,62 @@ ResourceUID::ID ResourceLoaderText::get_uid(Ref<FileAccess> p_f) {
 	return ResourceUID::INVALID_ID;
 }
 
+// Please call ResourceLoaderText::open() before calling this.
+// Returns empty string if the attached script could not be fetched.
+String ResourceLoaderText::get_attached_script_path(Ref<FileAccess> p_f) {
+	f = p_f;
+	stream.f = f;
+
+	ignore_resource_parsing = true;
+
+	while (next_tag.name == "ext_resource") {
+		if (!next_tag.fields.has("path")) {
+			return "";
+		}
+
+		if (!next_tag.fields.has("type")) {
+			return "";
+		}
+
+		if (!next_tag.fields.has("id")) {
+			return "";
+		}
+
+		String type = next_tag.fields["type"];
+
+		// Skip the tags until we find the "Script" tag
+		if (type == "Script") {
+			String path = next_tag.fields["path"];
+			String id = next_tag.fields["id"];
+
+			if (!path.contains("://") && path.is_relative_path()) {
```

Comment:
```suggestion
			if (path.is_relative_path()) {
```
I think that part is already checked by `is_relative_path()`.

Reply:
I copied this bit from `load()` Should I remove that part from both `load()` and `get_attached_script_path()`?

Reply:
I guess. The `is_relative_path()` checks for `:/`, which is to handle e.g. `C:/` on Windows, but this substring is also used in `res://`, so checking it twice is redundant.